src/main/java/com/shippo/model/LabelFileType.java

```diff
@@ -0,0 +1,5 @@
+package com.shippo.model;
+
+public enum LabelFileType {
+	PNG, PDF, PDF_4X6, ZPLII
```

Comment:
Just fyi this Github repo is public and the bitbucket repo link is private. Not that anyone will be able to access without credentials. But just be careful